metadata_layer.py

```diff
@@ -590,7 +590,7 @@ def set_default_rights_uri(self, data_source):
         for format in self.formats:
             if format.link:
                 link = format.link
-                if self.rights_uri == None and link.rel == Hyperlink.OPEN_ACCESS_DOWNLOAD:
+                if self.rights_uri in (None, RightsStatus.UNKNOWN) and link.rel == Hyperlink.OPEN_ACCESS_DOWNLOAD:
```

Comment:
I think this would only happen if something we're importing has a rights string but we don't recognize it, so then it should be fine to make it generic open access if there's an open access link.